index.html

```diff
@@ -3,47 +3,105 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>Democratic Delegate Count</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
+   
+
+
+    tr:hover td {
+        background-color: #f5f5f5 
+    }
```

Comment:
Note that this hover effect overrides the winner highlighting you set up below. It does so because the css rules deem this class more specific than the .winner styles. By simply removing the `td` from this class, though, the style would only apply to the row, and the highlighting would remain on the individual cells.